libs/utils/utils.js

```diff
@@ -794,8 +794,12 @@ export async function loadIms() {
     if (!window.adobeIMS?.isSignedInUser()) {
       getConfig().entitlements([]);
     }
+    // remove acomsis cookie that is used in Akamai, for SignOut event from ims.
+    window.adobeProfile?.addEventListener('adobeProfile:SignOut', () => {
```

Comment:
Hi @Blainegunn - Milo does not use the Profile Menu, the logic added here will never execute.